dipy/viz/actor.py

```diff
@@ -201,6 +201,139 @@ def copy(self):
     return image_actor
 
 
+def contour_actor(data, affine=None, levels=[50],
+            colors=[np.array([1.0, 0.0, 0.0])], opacities=[1]):
```

Comment:
PEP8: indentation level here doesn't match the line above (`colors` needs to be flush with `data`)